src/status_im/ui/screens/screens.cljs

```diff
@@ -214,7 +215,7 @@
     :insets    {:top false}
```

Comment:
when you just need to remove this , because it seems before some kind of animated custom header was used

Reply:
remove `:insets {:top false}`

Reply:
Oh alright, fixed it.